agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -536,6 +537,10 @@ func TestFluentdTag(t *testing.T) {
 	// tag was added in docker 1.9.0
 	RequireDockerVersion(t, ">=1.9.0")
 
+	if runtime.GOOS == "arm64" {
+		t.Skip()
```

Comment:
why are these tests skipped?

Reply:
Skipping this test for Arm as Fluentd docker container does not have official support for Arm.